cluster-controller/src/main/java/io/enmasse/barnabas/controller/cluster/operations/cluster/UpdateKafkaClusterOperation.java

```diff
@@ -35,8 +35,17 @@ public void execute(Vertx vertx, K8SUtils k8s, Handler<AsyncResult<Void>> handle
 
                 log.info("Updating Kafka cluster {} in namespace {}", name, namespace);
 
-                KafkaResource kafka = KafkaResource.fromConfigMap(k8s.getConfigmap(namespace, name));
-                ResourceDiffResult diff = kafka.diff(k8s.getStatefulSet(namespace, name));
```

Comment:
just asking ... is the fabric8 client to raise an exception when you are asking for a ConfigMap which doesn't exist ? Is it the same for statefulsets or for whatever resource you ask for ?

Reply:
Actually, thinking about it once again ... in this case it is rather an null pointer exception from KafkaResource.fromConfigMap(). So we can deal with it by checking the config map for null rather than wrapping it into try...chatch block. I will rework it.

Reply:
yeah it was my guess, why the `get()` or `getItems()` on the fabric8 client should not return null but an exception instead. I agree on the rework by checking null.

Reply:
@ppatierno Should be fixed now. Please have a second look.